### PR TITLE
Fix broken milestone munger

### PR DESCRIPTION
**Pull request by marun**
2 for 1: 
- Fix release milestone regex to support vx+.x+ (e.g. v1.10) instead of just vx+.x
- Use Ceil instead of Floor to compute days (e.g. grace period of 3 days should appear as 3 days instead of 2 in comments)

cc: @jdumars @jberkus 

Fixes: #7080

### CODE REVIEW

File: mungegithub/mungers/milestone-maintainer.go [hunk 1]
```diff
@@ -289,13 +289,8 @@ func dayPhrase(days int) string {
...
 }
 
-func durationToMinDays(duration time.Duration) string {
-	days := int(math.Floor(duration.Hours() / 24))
-	return dayPhrase(days)
-}
-
 func durationToMaxDays(duration time.Duration) string {
-	days := int(math.Floor(duration.Hours() / 24))
+	days := int(math.Ceil(duration.Hours() / 24))
```
Comment (BenTheElder) [inline mungegithub/mungers/milestone-maintainer.go]: LOL I remember seeing this code and asking @cjwagner if it was a bug
/shrug

**Comment by jdumars** [comment]
Thank you!!  

https://www.youtube.com/watch?v=bXtHwvp7jYE

**Comment by BenTheElder** [comment]
we'll still need to deploy this

**Comment by BenTheElder** [comment]
deploying

**Pull request merged**